docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Upgrade to Prefect 3
+description: Learn how to upgrade from Prefect 2 to Prefect 3.
+---
+
+Some patterns supported by Prefect 2 are no longer supported with Prefect 3, or are enabled differently. 
```

Comment:
Minor nit (non-blocking) - we may want a more positive spin on this line. "Prefect 3 brings new way to build and interact with your flows. This means that some patterns supported by Prefect 2 are no longer supported with Prefect 3, or are enabled differently."

This way we start with a carrot rather than an upgrade stick.